code/game/objects/effects/poster.dm

```diff
@@ -266,7 +265,7 @@
 
 	var/turf/user_drop_location = get_turf(user) //cache this so it just falls to the ground if they move. also no tk memes allowed.
 	if(!do_after(user, PLACE_SPEED, placed_poster, extra_checks = CALLBACK(placed_poster, TYPE_PROC_REF(/obj/structure/sign/poster, snowflake_wall_turf_check), src)))
-		to_chat(user, span_notice("The poster falls down!"))
+		balloon_alert(user, "interrupted!")
```

Comment:
Not necessary, since the do_after is already good enough visual feedback for this